Northstar.Client/mod/resource/northstar_client_localisation_polish.txt

```diff
@@ -14,7 +14,7 @@
 
 		"DIALOG_TITLE_INSTALLED_NORTHSTAR" "Dzięki za instalację Northstar!"
 		"AUTHENTICATION_AGREEMENT_DIALOG_TEXT" "Dla poprawnego działania Northstar potrzebne jest uwierzytelnienie z głównym serwerem. Potrzebne będzie wysłanie twojego tokena Origin do głównego serwera, nie będzie on przechowywany ani użyty do żadnych innych celów.
-Naciśnij "Tak" jeżeli wyrażasz zgodę. Wybór może zostać zmieniony w menu Modów."
```

Comment:
This should fix the issue, although if your want to keep the quotes around tak you can instead write `\"Tak\"`.

Reply:
by " Tak " i meant    Tak   without the quotes